src/lib.rs

```diff
@@ -148,6 +160,6 @@ pub mod pg_test {
 
     pub fn postgresql_conf_options() -> Vec<&'static str> {
         // return any postgresql.conf settings that are required for your tests
-        vec!["search_path = 'public, _prom_ext, ps_trace'"]
+        vec!["search_path = 'public, _prom_ext, ps_trace, _ps_trace'"]
```

Comment:
hmm actually why is the private schema needed in the tests? Does that mean users will also need it in their search_path for a good query experience (that shouldn't be the case).

Reply:
There isn't a really good reason for this, to be honest. Users shouldn't need this.
The reason was plan output contained schema name since it wasn't in the `search_path`. We decided that it'd be much nicer to change search path when we merge the commit where we make new schema structure/naming. The alternative would be to update all tests (which was deemed not as nice).

Reply:
Yeah, it's just convenience for unit tests. E2E doesn't rely on this.